### DISCUSSION TITLE
posts/670

### DISCUSSION THREAD

**Discussion by giscus** [kakaotech/comments#34] [Announcements]
# posts/670

들어가며
안녕하세요, 카카오 분산데이터베이스 조직에서 MongoDB를 운영하고 있...

https://tech.kakao.com/posts/670

**Comment by hakseon** [comment]
MongoDB를 관리하고 있는 관리자 입장에서 너무나 흥미로운 주제의 글이었던 것 같습니다.
덕분에 재밌게 보고 또 배우고 가네요 ㅎㅎㅎㅎ

**Comment by ZackGreinke** [comment]
정말 좋아요 백만개짜리 컬럼입니다..

**Comment by DongSeung2** [comment]
MongoDB를 도입하려고 하는데 많은 도움이 되었습니다.
